data/provider/src/test/java/tech/pegasys/teku/api/ChainDataProviderTest.java

```diff
@@ -388,6 +390,24 @@ public void getStateSyncCommittees_shouldRejectFarFutureEpoch() {
         .isCompletedExceptionallyWith(IllegalArgumentException.class);
   }
 
+  @Test
+  public void calculateRewards_shouldGetData() {
```

Comment:
this is good, what happens when we ask for a validator index that's not in the committee? can we demonstrate that in a unit test?

Reply:
Handled in next PR